src/main/java/hudson/maven/reporters/AbstractMavenJavadocArchiver.java

```diff
@@ -151,7 +148,7 @@ public String getDisplayName() {
             if (dir != null && new File(dir, "help-doc.html").exists())
                 return this.displayName;
             else
-                return hudson.tasks.Messages.JavadocArchiver_DisplayName_Generic();
+                return "Document";
```

Comment:
Unrelated, part of #127: the plugin was improperly using messages from Jenkins core, which are now `@Restricted`.